events/test_routes.py

```diff
@@ -174,6 +176,22 @@ def test_search_existing_event(self):
         self.assertEqual(len(data['events']), 1)
         self.assertEqual(data['num_events'], 1)
 
+    @patch('app.text_search_event_name', MagicMock(return_value=[VALID_DB_EVENT]))
+    def test_search_existing_event_uppercase(self):
+        """Search for an event that exists with different capitalization.
```

Comment:
If you've switched to a mock, you're no longer testing the behavior of Mongo; instead, you might want to [validate the arguments to `text_search_event_name`](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.Mock.call_args):

```python
aself.assertEqual(VALID_EVENT_NAME.upper(), app.text_search_event_name.call_args[0][1])
```

Reply:
Done in c6279ae